### PR TITLE
Testing: change schemaInfo to non-static

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by mfussenegger** [comment]
@aelesbao please review

File: sql/src/test/java/io/crate/analyze/InsertFromSubQueryAnalyzerTest.java [hunk 1]
```diff
@@ -63,9 +63,9 @@
 public class InsertFromSubQueryAnalyzerTest extends BaseAnalyzerTest {
 
     @Mock
-    private static SchemaInfo schemaInfo;
+    private SchemaInfo schemaInfo;
 
-    static class TestMetaDataModule extends MetaDataModule {
+    class TestMetaDataModule extends MetaDataModule {
```
Comment (aelesbao) [inline sql/src/test/java/io/crate/analyze/InsertFromSubQueryAnalyzerTest.java]: can it be private?

File: sql/src/test/java/io/crate/analyze/UpdateAnalyzerTest.java [hunk 2]
```diff
@@ -82,14 +82,14 @@
         .addPrimaryKey("o.x")
         .build();
 
-    static TableInfo partedGeneratedColumnTableInfo;
-    static TableInfo nestedPartedGeneratedColumnTableInfo;
+    TableInfo partedGeneratedColumnTableInfo;
+    TableInfo nestedPartedGeneratedColumnTableInfo;
```
Comment (aelesbao) [inline sql/src/test/java/io/crate/analyze/UpdateAnalyzerTest.java]: can these be private too?

**Comment by aelesbao** [comment]
lgtm otherwise

**Pull request merged**